src/styles/blocks/main/card/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $background-color-card;
+  width: 198px;
+  height: 406px;
+  transition: transform 300ms;
+
+  &__logo {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__info {
+    margin: 36px 16px 16px;
+    color: #060b35;
+
+    &-title {
+      width: 166px;
```

Comment:
I added nav__link--is-active but now Tests doesn`t see the class is-active in the task. The cards go to the left side because we have a header, does it need to add styles for the header on the tablet or mobile version? Pls chack))